vercel/turbo
# Adding Vercel Toolbar to docs.

### Description

To create a better feedback loop with the community, we can enable comments on our documentation.

This PR enables the Vercel Toolbar for local branches and production. On previews, the toolbar will always be present.

In a typical product, you wouldn't display your toolbar for any site visitor to use on production. However, I'm doing an overhaul of our documentation so I'm interested in collecting as much feedback as possible. Depending on the quantity and quality of comments we get, we may or may not choose to keep Toolbar active on production long term.

### Testing Instructions

Click the new button in the top right of the preview to enable the Vercel Toolbar. Leave a comment and resolve it! Fun!

Note that this may be awkward to test on a preview deployment since previews always have the toolbar. You will likely want to take a look at the behavior locally by pulling the branch and using `cd docs && pnpm build && pnpm start`.
